src/main/java/com/ecore/roles/service/impl/TeamsServiceImpl.java

```diff
@@ -3,26 +3,24 @@
 import com.ecore.roles.client.TeamsClient;
 import com.ecore.roles.client.model.Team;
 import com.ecore.roles.service.TeamsService;
-import org.springframework.beans.factory.annotation.Autowired;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.UUID;
 
 @Service
+@RequiredArgsConstructor
 public class TeamsServiceImpl implements TeamsService {
 
     private final TeamsClient teamsClient;
 
-    @Autowired
```

Comment:
This constructor can be replaced with @RequiredArgsConstructor lombok annotation in order to become the code cleaner